app/helpers/application_helper.rb

```diff
@@ -1,4 +1,12 @@
 module ApplicationHelper
+
+  def liked_by(tweet)
+    liked_by = []
+    tweet.likes.each do |user|
+      liked_by << User.find(user.user_id)
+    end
+  end
```

Comment:
This should be a model method, please refer the usage of application helper again.